
from CustomExceptions.base_exceptions import BaseException

class FailedFatchingError(BaseException):
    ...